### PR TITLE
Update bundle_zh_CN.properties

### PR DESCRIPTION
**Pull request by Cyl18**

### CODE REVIEW

**Comment by laohuaji233** [comment]
实话说地编的过滤器有的你确定这样翻译？比如noise？

File: core/assets/bundles/bundle_zh_CN.properties [hunk 1]
```diff
@@ -64,21 +64,21 @@ researched = [LIGHT_GRAY]{0}己研究。
...
-server.kicked.kick = 你被踢出服务器了！
+server.kicked.kick = 你被踢出了服务器！
 server.kicked.serverClose = 服务器已关闭。
-server.kicked.vote = You have been vote-kicked. Goodbye.
+server.kicked.vote = 你被投票踢出了服务器. 再见.
 server.kicked.clientOutdated = 客户端过旧，请更新你的游戏。
 server.kicked.serverOutdated = 服务器过旧，请联系房主升级服务器。
 server.kicked.banned = 你在这个服务器上被拉入黑名单了。
-server.kicked.typeMismatch = This server is not compatible with your build type.
+server.kicked.typeMismatch = 服务器与你的客户端构建类型不兼容.
```
Comment (laohuaji233) [inline core/assets/bundles/bundle_zh_CN.properties]: 指的应该是不稳定测试版

File: core/assets/bundles/bundle_zh_CN.properties [hunk 2]
```diff
@@ -518,10 +518,10 @@ setting.mutemusic.name = 静音
 setting.sfxvol.name = 音效音量
 setting.mutesound.name = 静音
 setting.crashreport.name = 发送匿名崩溃报告
-setting.savecreate.name = Auto-Create Saves
+setting.savecreate.name = 自动存档
```
Comment (laohuaji233) [inline core/assets/bundles/bundle_zh_CN.properties]: 自动创建存档

File: core/assets/bundles/bundle_zh_CN.properties [hunk 3]
```diff
@@ -867,8 +867,8 @@ unit.eradicator.name = 根除者
 unit.lich.name = 尸鬼
 unit.reaper.name = 死神
 tutorial.next = [lightgray]<点击以继续>
-tutorial.intro = 你进入了[scarlet] Mindustry 教程[]。\n[accent]采集铜矿[]以开始。点击附近的一处铜矿。\n\n[accent]{0}/{1} 铜
-tutorial.drill = 手动采矿效率低。\n[accent]钻头[]可以自动采矿。\n放一个在铜矿上吧。\n点击右下角的钻头菜单。\n选择[accent]机械钻头[]。\n单击将其放置在铜矿上。\n[accent]右键单击[]来停止。
+tutorial.intro = 你进入了[scarlet] Mindustry 教程[]。\n[accent]采集铜矿[]来开始。点击附近的一处铜矿。\n\n[accent]{0}/{1} 铜
+tutorial.drill = 手动采矿效率较低。\n[accent]钻头[]可以自动采矿。\n放一个在铜矿上吧。\n点击右下角的钻头菜单。\n选择[accent]机械钻头[]。\n单击将其放置在铜矿上。\n[accent]右键单击[]来停止。
 tutorial.drill.mobile = 手动采矿效率低。\n[accent]钻头[]可以自动采矿。\n点击右下角的钻头菜单。\n选择[accent]机械钻头[]。\n点击将其放在铜矿上，点击[accent]对号[]来确定。\n点击[accent]叉号[]来取消。
```
Comment (laohuaji233) [inline core/assets/bundles/bundle_zh_CN.properties]: 手机端和电脑不一样

**Comment by Anuken** [comment]
Seems to have been superseded by @laohuaji233's PRs.

**Pull request closed without merging**